Trio/Sources/Modules/NightscoutConfig/View/NightscoutUploadView.swift

```diff
@@ -35,7 +35,7 @@ struct NightscoutUploadView: View {
                 VStack(alignment: .leading, spacing: 10) {
                     Text("Default: OFF").bold()
                     Text(
-                        "The Upload Treatments toggle enables uploading of the following data sets to your connected Nightscout URL:"
+                        "This toggle enables uploading of the following data sets to your connected Nightscout URL:"
```

Comment:
I'd suggest not to change this, as the pre-change text is not necessarily wrong, and part of many localizations already.